### DISCUSSION TITLE
Overflow-hidden doesn't work around the rounded corners when using animate in safari

### DISCUSSION THREAD

**Discussion by kevin-dsouza** [tailwindlabs/tailwindcss#5675] [Help] [answered]
This code doesn't  in safari. Not really a Tailwind bug, more of a safari bug. Wondering if there's a workaround in tailwind.

```
<div class="h-40 w-40 border border-red-100 rounded-3xl m-10 overflow-hidden">
  <div class="bg-red-800 rounded-full h-12 w-12 animate-ping"></div>
</div>
```

![image](https://user-images.githubusercontent.com/2907754/135700276-374392aa-2175-449c-9a5b-dbb6df7cd194.png)

**Comment by simonswiss** [comment, marked as answer]
Heh, weird bug indeed!

_Please_ don't ask me _why_ it works, but I found this solution that, when added to the parent (with the `overflow-hidden` class), works a charm on Safari:

```css
 -webkit-mask-image: -webkit-radial-gradient(white, black);
```

https://gist.github.com/ayamflow/b602ab436ac9f05660d9c15190f4fd7b#file-gistfile1-css

Here's your updated Tailwind Play example with a custom utility added (and appropriately named `whaaaaat`):

https://play.tailwindcss.com/WIbHfEnty9?file=css

It works nicely in Safari!

![image](https://user-images.githubusercontent.com/485747/137043311-67e71b11-5c5f-46d4-aac3-57e75e8f3b70.png)

Again, don't ask me _why_ 😅

**Reply by kevin-dsouza** [reply]
Thank you @simonswiss. Your solution did work like a charm! Also love your vids!!

**Reply by dimaswntech** [reply]
thanks from me too

**Comment by leo** [comment]
You might want to add the [isolate](https://tailwindcss.com/docs/isolation) utility instead!

I believe it has the same effect (a new stacking context), but it‘s cleaner.

**Reply by madwyn** [reply]
Amazing, this solve my problem.

I needed to create a rounded mask for a Gatsby 'StaticImage' which doesn't support props.

```HTML
<div className="overflow-hidden rounded-full isolate">
                    <StaticImage
                        src='../images/example.jpg'
                        loading="eager"
                        height={512}
                        quality={95}
                        formats={["auto", "webp", "avif"]}
                    />
</div>
```
